keywords.robot

```diff
@@ -74,6 +74,9 @@ Login To Windows
     IF    '${DUT_CONNECTION_METHOD}' == 'Telnet'
         Set Test Variable    ${DUT_CONNECTION_METHOD}    SSH
     END
+    IF    '${DUT_CONNECTION_METHOD}' == 'Telnet'
+        Set Global Variable    ${DUT_CONNECTION_METHOD}    SSH
+    END
```

Comment:
Please put that in a single IF:

```
    IF    '${DUT_CONNECTION_METHOD}' == 'Telnet'
        Set Test Variable    ${DUT_CONNECTION_METHOD}    SSH
        Set Global Variable    ${DUT_CONNECTION_METHOD}    SSH
    END
```

Also if we do `Set Global Variable`, is `Set Test Variable` still needed?

Reply:
I'm not sure, those are actually not my changes.

Reply:
Done per suggestion.

Reply:
Looking at this fragment of code, it checks whether connection method is telnet and if it is, it switches connection method to SSH. The next conditional checks whether connection method is SSH. So we could just do something like:
```
    IF    '${DUT_CONNECTION_METHOD}' == 'SSH' or '${DUT_CONNECTION_METHOD}' == 'Telnet'
        Login To Windows Via SSH    ${DEVICE_WINDOWS_USERNAME}    ${DEVICE_WINDOWS_PASSWORD}
    ELSE
        Fail    Login to Windows not supported. DUT_CONNECTION_METHOD must be set to SSH.
    END
```

Reply:
But these are in your commit: 14f28b7312c7a7b4f26c8530382f93a7f94125ff

I see you have merged develop into your branch, a better idea is to use rebase when updating your branch. We are in favor of avoiding merge commits this way.